android/uhabits-core/src/test/java/org/isoron/uhabits/core/io/ImportTest.java

```diff
@@ -81,7 +81,7 @@ public void testRewireDB() throws IOException
 
         assertThat(habitList.size(), equalTo(3));
 
-        Habit habit = habitList.getByPosition(0);
+        Habit habit = habitList.getByPosition(1);
```

Comment:
Same as the previous comment.

Reply:
Also same answer, the default secondary ordering is by name, so the insertion ordering is no longer there.